README.md

```diff
@@ -1,2 +1,24 @@
 # docker-images
 🌅 Docker images for the Nebari platform
+
+
```

Comment:
This readme went straight to the pinning... as an external contributor, I want to know the following:

1. The CI status - TODO: add badge for the CI
1. What is in this repo - i.e. the purpose of the images
2. How do I get started? what dependencies do I need? 
3. Step by step instructions to build the images because more than likely I will copy and paste in whichever order the instructions come in
4. Can I use them locally? if so how do I do this?
5. What is the license?
6. How do I report issues?
7. Where are the images pushed?

Inspiration:
- https://github.com/jupyterhub/repo2docker
- https://github.com/openshift/source-to-image
- https://github.com/jupyter/docker-stacks

Reply:
Added most of these items to the README 👍